packages/core/src/js/feedback/ScreenshotButton.tsx

```diff
@@ -8,20 +8,23 @@ import { defaultScreenshotButtonConfiguration } from './defaults';
 import { defaultScreenshotButtonStyles } from './FeedbackWidget.styles';
 import { getTheme } from './FeedbackWidget.theme';
 import type { ScreenshotButtonProps, ScreenshotButtonStyles, ScreenshotButtonTextConfiguration } from './FeedbackWidget.types';
+import { hideScreenshotButton, showFeedbackWidget, showScreenshotButton } from './FeedbackWidgetManager';
```

Comment:
The cyclic dep failure is fixed in https://github.com/getsentry/sentry-react-native/pull/4718